src/ds_caselaw_utils/test_courts.py

```diff
@@ -67,6 +67,60 @@ def test_loads_court_by_param(self):
         repo = CourtsRepository(data)
         self.assertEqual("Court 2", repo.get_by_param("court2").name)
 
+    def test_returns_listable_courts(self):
+        data = [
```

Comment:
can we factor this out into a fixture rather than creating it twice?
(two groups: one tribunals, one courts, both with a listable and non-listable entity?)